docs/usage/security/jwt.rst

```diff
@@ -33,6 +33,6 @@ OAuth2 Bearer Password Flow
 This backend inherits from the :class:`JWTCookieAuth <litestar.contrib.jwt.JWTCookieAuth>` backend. It works similarly to
```

Comment:
```suggestion
This backend inherits from the :class:`JWTCookieAuth <litestar.security.jwt.JWTCookieAuth>` backend. It works similarly to
```

do we need to change these also?

Reply:
We do. I wonder why sphinx isn't complaining about this?